Extensions/fullScreen.js

```diff
@@ -121,6 +121,121 @@ const translations = {
         tvBtnDesc: "TV Mode Display",
         fullscreenBtnDesc: "Full Screen",
     },
+    const translations = {
+    "de--DE": {
+        langName: "Deutsch",
+        context: {
+            queue: "Wiedergabe aus der Warteschlange",
+            track: "Titel spielen",
+            album: "Abspielen von Album",
+            artist: "Abspielen von Künstler",
+            playlist: "Abspielen von Playlist",
+            playlistFolder: "Wiedergabe aus dem Playlist-Ordner",
+            search: "Wiedergabe aus Suche",
+            searchDest: "Lieder",
+            collection: "Abspielen aus Sammlung",
+            likedSongs: "Lieblingssongs",
+            trackRadio: "Abspielen von Titel Radio",
+            artistRadio: "Abspielen von Künstler Radio",
+            albumRadio: "Abspielen von Alben Radio",
+            playlistRadio: "Abspielen von Playlist Radio",
+        },
+        upnext: "ALS NÄCHSTES",
+        unknownArtist: "Künstler (nicht verfügbar)",
+        settings: {
+            switchToTV: "Wechseln zu TV-Modus",
+            switchToFullscreen: "Wechseln zu Standardmodus",
+            tvModeConfig: "Konfiguration des TV-Modus",
+            fullscreenConfig: "Vollbildmodus konfiguration",
```

Comment:
I'd instead use "Einstellungen des Vollbildmodis" or something like that, "Konfiguration" is a noun, and thus has to be capitalized